Packs/Jira/Integrations/JiraV3/README.md

```diff
@@ -688,6 +688,39 @@ There is no context output for this command.
 >|---|---|---|---|
 >| https:<span>//</span>api.atlassian.com/ex/jira/1234/rest/api/3/attachment/16505 | dummy.pdf | 16505 | PROJECTKEY-31 |
 
+
+### jira-issue-delete-file
+
+
+***
+Deletes an attachment from an issue.
+
+Scope: `write:jira-work`
+
+#### Base Command
+
+`jira-issue-delete-file
+`
+
+#### Input
+
+| **Argument Name** | **Description** | **Required** |
+| --- | --- | --- |
+| attachment_id | The attachment ID. | Required |
+
+#### Context Output
+
+There is no context output for this command.
+
+#### Command example
+
+```!jira-issue-delete-file attachment_id=16777```
```

Comment:
Lets change the `attachment_id` value to "PROJECTKEY" (see other examples from other commands).
Please also change it in `Human Readable Output` accordingly.